tfhe/src/high_level_api/keys/inner.rs

```diff
@@ -164,6 +216,8 @@ impl IntegerServerKey {
 #[derive(Clone, serde::Serialize, serde::Deserialize)]
 pub struct IntegerCompressedServerKey {
     pub(crate) key: crate::integer::CompressedServerKey,
+    pub(crate) compression_keys:
+        Option<(CompressedGlweCompressionKey, CompressedGlweDecompressionKey)>,
```

Comment:
why do we have both here if we don't necessarily have both in the normal ServerKey ?

Reply:
Separated